.github/workflows/build.yml

```
@@ -35,6 +35,26 @@ jobs:
         id: date
         run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
 
+      - name: Setup apptainer
+        run: |
+          sudo add-apt-repository -y ppa:apptainer/ppa
+          sudo apt-get update
+          sudo apt install -y apptainer squashfuse
+
+      # See https://github.com/apptainer/apptainer/pull/2262
+      - name: Disable apparmor namespace restrictions for apptainer
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:10:25: Double quote to prevent globbing and word splitting

(shellcheck)